inc/languages/english/online.lang.php

```diff
@@ -66,7 +66,7 @@
 $l['editing_signature'] = "<a href=\"usercp.php?action=editsig\">Editing Signature</a>";
 $l['changing_avatar'] = "<a href=\"usercp.php?action=avatar\">Changing Avatar</a>";
 $l['viewing_subscriptions'] = "Viewing <a href=\"usercp.php?action=subscriptions\">Thread Subscriptions</a>";
-$l['editing_pad'] = 'Editing <a href="usercp.php">Personal Pad</a>';
+$l['editing_pad'] = "Editing <a href=\"usercp.php\">Personal Pad</a>";
```

Comment:
Probably even anchor tags altogether but not so sure about these. Maybe all HTML tags should be in templates, which might hard to decide because of how the current system works, but once the new template system (more optimized) is functional maybe we can review this.